ucdlib-special/includes/collection.php

```diff
@@ -136,6 +136,24 @@ public function set_context($context){
     if ( $context['post']->post_type !== $this->slug ) return $context;
     $p = $context['post'];
     $context['config'] = $this->config;
+
+    // add breadcrumbs
+    $manuscriptLanderCrumbs = [
+      [
+        'link' => '/',
+        'title' => 'Home'
+      ],
+      [
+        'link' => '/archives-and-special-collections',
+        'title' => 'Archives and Special Collections'
+      ], 
+      [
+        'link' => '/archives-and-special-collections/manuscripts',
+        'title' => 'Manuscripts'
+      ]
+    ];
+    $context['breadcrumbs'] = array_merge($manuscriptLanderCrumbs, [['title' => $context['post']->title]]);
```

Comment:
what happens if it is a university archive and not a manuscript?

Reply:
Ah that's right. What would the url end up being for university archives?